src/spec/integration/product-type-importer.spec.coffee

```diff
@@ -0,0 +1,108 @@
+{expect} = require 'chai'
+Promise = require 'bluebird'
+ProductTypeImporter = require '../../lib/product-type-import'
+{SphereClient} = require 'sphere-node-sdk'
+{ProjectCredentialsConfig} = require 'sphere-node-utils'
+
+errMissingCredentials = 'Missing configuration in env variables'
+
+argv =
+  projectKey: process.env.SPHERE_PROJECT_KEY
+  logSilent: true
+
+testProductType = {
+  "name": "top_and_shirts",
+  "description": "Tops & Shirts",
+  "attributes": [
+    {
+      "name": "designer",
+      "label": {
+        "de": "Designer",
+        "en": "designer"
+      },
+      "type": {
+        "name": "text"
+      },
+      "attributeConstraint": "SameForAll",
+      "isRequired": false,
+      "isSearchable": false,
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "materialComposition",
+      "label": {
+        "de": "Zusammensetzung Material",
+        "en": "material composition"
+      },
+      "type": {
+        "name": "ltext"
+      },
+      "attributeConstraint": "None",
+      "isRequired": false,
+      "isSearchable": false,
+      "inputHint": "MultiLine"
+    }
+  ]
+}
+
+describe 'ProductTypeImporter', ->
+  importer = null
+  sphereClient = null
+
+  before ->
```

Comment:
I switched it to mocha which is working with Promises too - so it uses promise instead of done callback